### PR TITLE
add c3pm binary installation doc

### PR DESCRIPTION
**Pull request by Codelax**
- [x] Linux
- [x] Macos
- [x] Windows

### CODE REVIEW

File: docs/getting_started/install.md [hunk 1]
```diff
@@ -3,14 +3,84 @@ id: install
...
+
+We provide prebuilt binaries directly in [github release](https://github.com/c3pm-labs/c3pm/releases/latest)
+
+You can download packages there or use our repos when available
+
+
+### Linux
+
+:::caution
+No c++ compiler is being installed alongside c3pm, this is left to user
```
Comment (chloe-opteameo) [inline docs/getting_started/install.md]:
```suggestion
No c++ compiler is being installed alongside c3pm, this is left to the user
```

File: docs/getting_started/install.md [hunk 2]
```diff
@@ -3,14 +3,84 @@ id: install
...
+```bash
+yum install ./c3pm_0.0.1_linux_amd64.rpm    # CentOS
+dnf install ./c3pm_0.0.1_linux_amd64.rpm    # Fedora/RHEL
+zypper install ./c3pm_0.0.1_linux_amd64.rpm # OpenSUSE
+```
+
+</TabItem>
+<TabItem value="manually">
+
+Before c3pm, you should install cmake and a c++ compiler
```
Comment (chloe-opteameo) [inline docs/getting_started/install.md]:
```suggestion
Before installing c3pm, you should install CMake and a c++ compiler
```

File: docs/getting_started/install.md [hunk 3]
```diff
@@ -3,14 +3,84 @@ id: install
...
+ls
+#> LICENSE.txt  README.md  c3pm_0.0.1_Linux_x86_64.tar.gz  ctpm
+```
+</TabItem>
+</Tabs>
+
+
+### MacOS
+
+C3PM provide a tap to install and update it easily
```
Comment (chloe-opteameo) [inline docs/getting_started/install.md]:
```suggestion
C3PM provides a tap to install and update it easily
```

File: docs/getting_started/install.md [hunk 4]
```diff
@@ -3,14 +3,84 @@ id: install
...
+brew install c3pm
+ctpm -v
+```
+
+### Windows
+
+We do not yet have repositories to easily install c3pm on windows.
+
+You should download windows archive in [github release](https://github.com/c3pm-labs/c3pm/releases/latest).
+Installation of cmake and a c++ compiler is left to user.
```
Comment (chloe-opteameo) [inline docs/getting_started/install.md]:
```suggestion
Installation of CMake and a c++ compiler is left to the user.
```

File: docs/getting_started/install.md [hunk 5]
```diff
@@ -3,14 +3,84 @@ id: install
...
+
+```
+brew tap c3pm-labs/c3pm
+brew install c3pm
+ctpm -v
+```
+
+### Windows
+
+We do not yet have repositories to easily install c3pm on windows.
```
Comment (angela-hubner) [inline docs/getting_started/install.md]: Windows instead of windows

**Review by angela-hubner — CHANGES_REQUESTED** [review]

File: docs/getting_started/install.md (hunk 5 above)
Comment (chloebourbion) [reply docs/getting_started/install.md]:
```suggestion
We do not yet have repositories to easily install c3pm on Windows.
```

**Review by chloebourbion — APPROVED** [review]

**Review by chloebourbion — APPROVED** [review]

**Review by angela-hubner — APPROVED** [review]

**Pull request merged**